Dockerfile

```diff
@@ -285,16 +285,17 @@ RUN \
 FROM sdk-musl as sdk-musl-openssl
 USER builder
 
-ARG OPENSSLVER="3.0.2"
-ARG OPENSSLREV="4"
+ARG OPENSSLVER="3.0.5"
+ARG OPENSSLREV="1"
 
 WORKDIR /home/builder
 COPY ./hashes/openssl ./hashes
 RUN \
   sdk-fetch hashes && \
   rpm2cpio openssl-${OPENSSLVER}-${OPENSSLREV}.*.src.rpm | cpio -idmv && \
-  tar xf openssl-${OPENSSLVER}-hobbled.tar.gz && \
+  tar xf openssl-${OPENSSLVER}-hobbled.tar.xz && \
   mv openssl-${OPENSSLVER} openssl && \
+  grep -Fwl '<sys/sdt.h>' *.patch | xargs rm && \
```

Comment:
That makes sense. I wanted to avoid having to adjust this every time the upstream patch numbers changed, but that's not a big deal, and I agree that it's better than the alternative of silently deleting a patch we might care about.